fluent-tests/src/test/java/com/azure/mgmttest/RuntimeTests.java

```diff
@@ -304,6 +308,22 @@ private void testAdvisor(StorageAccount storageAccount) {
         Assertions.assertEquals(countBeforeSuppress, countAfterDelete);
     }
 
+    @Test
+    @Disabled("live test")
+    public void testStorageWithPipeline() {
+        AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+        HttpClient httpClient = HttpClient.createDefault();
+        HttpPipeline httpPipeline = new HttpPipelineBuilder()
+                .httpClient(httpClient)
+                .policies(new ArmChallengeAuthenticationPolicy(
+                        new EnvironmentCredentialBuilder().build(),
+                        profile.getEnvironment().getManagementEndpoint() + "/.default"),
```

Comment:
Minimal pipeline requires `ArmChallengeAuthenticationPolicy`, with credential and scope.